README.md

```diff
@@ -0,0 +1,3 @@
+# Flashcards
+
+Flashcards is a CLI based flashcard based tutor that can be used to learn topics in a card based manner, with a phrase in the front, and the answer at the back of the card.
```

Comment:
Like how to run the app```suggestion
Flashcards is a CLI based flashcard based tutor that can be used to learn topics in a card based manner, with a phrase in the front, and the answer at the back of the card.
```
Flashcards is a CLI based flashcard based tutor that can be used to learn topics in a card based manner, with a phrase in the front, and the answer at the back of the card. To run this app run xyz....